pyproject.toml

```diff
@@ -95,6 +95,7 @@ select = [
     "ICN", # import conventions
     "RUF", # ruff
     "ISC001", # single-line-implicit-string-concatenation
+    "TID", # flake8-tidy-imports
```

Comment:
This promotes more explicit imports (`from virtualship.models` import ...` as opposed to `from ..models import ...`). See [here](https://docs.astral.sh/ruff/rules/#flake8-tidy-imports-tid) for more info